app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/infowindow/MarkerManager.kt

```diff
@@ -0,0 +1,154 @@
+package com.mapbox.maps.testapp.examples.markersandcallouts.infowindow
+
+import android.view.View
+import com.mapbox.geojson.Point
+import com.mapbox.maps.MapView
+import com.mapbox.maps.ScreenCoordinate
+import com.mapbox.maps.plugin.annotation.annotations
+import com.mapbox.maps.plugin.annotation.generated.OnPointAnnotationClickListener
+import com.mapbox.maps.plugin.annotation.generated.PointAnnotation
+import com.mapbox.maps.plugin.annotation.generated.PointAnnotationManager
+import com.mapbox.maps.plugin.annotation.generated.createPointAnnotationManager
+import com.mapbox.maps.plugin.gestures.OnMapClickListener
+import com.mapbox.maps.plugin.gestures.addOnMapClickListener
+import com.mapbox.maps.plugin.gestures.removeOnMapClickListener
+import com.mapbox.maps.viewannotation.OnViewAnnotationPositionUpdatedListener
+import com.mapbox.maps.viewannotation.viewAnnotationOptions
+import java.util.concurrent.CopyOnWriteArrayList
+import kotlin.math.abs
+
+/**
+ * Manager class helping to control the [Marker]s.
+ */
+class MarkerManager(
```

Comment:
would it make sense to move this class or similar abstraction to the view annotation(or as a separate plugin)?

Reply:
no, I've mentioned this point in PR description, we're leaving this as reference implementation for now 😉